cp-quickstart/start-docker-cloud.sh

```diff
@@ -5,95 +5,162 @@
 # To avoid unexpected charges, carefully evaluate the cost of resources before launching the script and ensure all resources are destroyed after you are done running it.
 #########################################
 
+NAME=`basename "$0"`
 
 # Source library
-. ../utils/helper.sh
+source ../utils/helper.sh
+
+check_jq \
+  && print_pass "jq found"
 
 check_ccloud_version 1.0.0 \
-  && print_pass "ccloud version ok" \
-  || exit 1
+  && print_pass "ccloud version ok"
+
 check_ccloud_logged_in \
-  && print_pass "logged into ccloud CLI" \
-  || exit 1
-
-echo
-wget -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-cloud/docker-compose.yml
-
-echo ====== Confirm
-prompt_continue_cloud_demo || exit 1
-read -p "Do you acknowledge this script creates a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
-echo
-if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
-
-echo
-echo ====== Create new Confluent Cloud stack
-cloud_create_demo_stack true || exit 1
-SERVICE_ACCOUNT_ID=$(ccloud kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4;}')
+  && print_pass "logged into ccloud CLI" 
+
+print_pass "Prerequisite check pass"
+
+[[ -z "$AUTO" ]] && {
+  printf "\n====== Confirm\n\n"
+  prompt_continue_cloud_demo || exit 1
+  read -p "Do you acknowledge this script creates a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
+  if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
+	printf "\n"
+} 
+
+printf "\nFor your reference the demo will highlight some commands in "; print_code "code format"
+printf "\nFor example:\n"
```

Comment:
@rspurgeon I have run this example 3 times now, and all 3 times I tripped on this output because my eyes were drawn to the red x and I thought at first something failed.  Also, the green checkmark and red x are self-explanatory that it seems unnecessary to tell the user what it means.  So WDYT about removing these 4 lines (33-36)

Reply:
k, removed